sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -563,6 +563,10 @@
                                 "label": "Undergraduate Certificate",
                                 "value": "undergraduateCertificate"
                             },
+                            {
+                                "label": "Undergraduate Citation",
+                                "value": "undergraduateCitation"
+                            },
                             {
```

Comment:
`onlyAvailableItems` seems to be false for this component which needs to be set true. May not be part of PR but important.

Reply:
I changed that to true.